src/Controls/src/Core/SolidColorBrush.cs

```diff
@@ -53,7 +53,7 @@ public override bool Equals(object obj)
 		/// <include file="../../docs/Microsoft.Maui.Controls/SolidColorBrush.xml" path="//Member[@MemberName='GetHashCode']/Docs/*" />
 		public override int GetHashCode()
 		{
-			return -1234567890 + Color.GetHashCode();
+			return -1234567890 + (Color?.GetHashCode() ?? 0);
```

Comment:
Some concern with this code (not the fix)
- -1234567890 isn't prime
- this could overflow (well underflow)
- it's missing the unchecked context
- could be more efficient by using `^` instead of `+`

Reply:
@StephaneDelcroix I’ve made the changes based on your suggestions. Let me know if you have any further feedback.